R/rater_functions.R

```diff
@@ -61,7 +68,11 @@ meanKappa <- function( dat , type = c("Cohen", "BrennanPrediger"), weight = "unw
   # weight.mean ... = T, if agreement is weighted by number of rater subjects,
   #            = F, if it is averaged among all rater pairs
   dat  <- eatTools::makeDataFrame(dat)
+  checkmate::assert_character(weight, len = 1)
+  checkmate::assert_logical(weight.mean, len = 1)
   type <- match.arg(type)
```

Comment:
ok, so when I used assert_choice (similar to subset, but allows only scalar input) it doesn't work, but with the `match.arg` line it does, so I'll leave it like that?

Reply:
Yes please leave match.arg